hardhat.config.ts

```diff
@@ -38,4 +42,12 @@ module.exports = {
             },
         },
     },
+    typechain: {
+        outDir: "types",
+        target: "ethers-v5",
+        // should overloads with full signatures like deposit(uint256) be generated always, even if there are no overloads?
+        alwaysGenerateOverloads: false,
+        // optional array of glob patterns with external artifacts to process (for example external libs from node_modules)
+        externalArtifacts: ["externalArtifacts/*.json"],
+    },
```

Comment:
Just taking the defaults from here https://github.com/dethcrypto/TypeChain/tree/master/packages/hardhat